bsyncviewer/testsuitelib/energyElements.sch

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<schema xmlns="http://purl.oclc.org/dsdl/schematron" queryBinding="xslt2">
+<schema xmlns="http://purl.oclc.org/dsdl/schematron" queryBinding="xslt1">
```

Comment:
lxml only supports xslt1 and will fail if we use xslt2 for queryBinding

Reply:
I can't believe that's all i would have had to change to get lxml to work!  :)